### PR TITLE
Block user with devise and omni via facebook

### PR DESCRIPTION
**Pull request by thaotonto**
screen shot 2018-08-23 at 9 34 15 am

### CODE REVIEW

**Comment by noobsi** [comment]
ok

File: app/controllers/users/omniauth_callbacks_controller.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class Users::OmniauthCallbacksController < Devise::OmniauthCallbacksController
+  def facebook
```
Comment (DuongNT1512) [inline app/controllers/users/omniauth_callbacks_controller.rb]: chưa đúng restful nhé em

File: app/models/order.rb [hunk 2]
```diff
@@ -9,6 +9,7 @@ class Order < ApplicationRecord
   validates :user, presence: true
   validates :screening, presence: true
   validates :paid, presence: true
+  scope :paid, ->{where paid: :paid}
```
Comment (DuongNT1512) [inline app/models/order.rb]: k cần viết scope nhé

File: app/models/user.rb [hunk 3]
```diff
@@ -60,4 +69,33 @@ def handle type
...
+      user = find_by email: email
+      if user&.uid
+        user
+      elsif user && !user.uid
+        user.uid = auth.uid
+        user.provider = auth.provider
+        user.save!
+        user
+      else
+        user = User.new(email: email, password: Devise.friendly_token[0, 20],
```
Comment (DuongNT1512) [inline app/models/user.rb]: remove ()

**Pull request merged**